dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcClientProvider.java

```diff
@@ -22,6 +22,7 @@
  */
 public class JdbcClientProvider implements DbClientProvider {
 
+    // Name of this JDBC DB client provider and also JDBC database URL prefix.
```

Comment:
This should not be used as a prefix for URL. This is the DbClient provider id.

Reply:
Separated those String constants to be used just for a single purpose.